cmd/create/execution.go

```diff
@@ -0,0 +1,123 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/lyft/flytectl/cmd/config"
+	cmdCore "github.com/lyft/flytectl/cmd/core"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+)
+
+const (
+	executionShort = "Create execution resources"
+	executionLong  = `
+Create the executions for given launchplan in a project and domain.
+Currently the launchplan and where the execution needs to be launched , need to belong to same domain and project.
+
+There are three steps in generating an execution.
+
+- Generate the execution param yaml file containing in the default input parameters for the launch plan.
+- If the default parameters exist then you can modify the default param values in the generated yaml file.
+- Run the execution passing in the generated yaml file.
+
+The following command would generate core.basic.lp.go_greet.inputs.yaml file by using the launch plan provided in argument.
+This would use the latest version of the launch plan.
+::
+
+ bin/flytectl create execution core.basic.lp.go_greet  -d development  -p flytesnacks -g
+
+The generated file would look similar to this
+
+.. code-block:: yaml
+
+	inputs:
+	  am: false
+	  day_of_week: ""
+	  number: 0
+
+The generated file can be modified to change the parameter values and can be passed to command for execution
+::
+
+ bin/flytectl create execution core.basic.lp.go_greet -f core.basic.lp.go_greet.inputs.yaml  -d development  -p flytesnacks
```

Comment:
so one problem - In flyte for an execution there are 2 `project and domain`s. One can be the source project/domain where the lauchplan/task is defined and the other can be the target project/domain where the execution is actually triggered.

So these should be the options
```bash
flytectl create execution --project x --domain y --launchPlan lp --targetProject p --targetDomain d
```
this will create an execution in "p-d" for launchplan in "x-y"

Absence of targetProject/targetDomain, we could use the same project/domain
```bash
flytectl create execution --project x --domain y --launchPlan lp
```
this will create an execution in "x-y" for launchplan in "x-y"

ALso for Task
```bash
flytectl create execution --project x --domain y --task task --targetProject p --targetDomain d
```

Reply:
Done